src/components/Player/Player.tsx

```diff
@@ -283,6 +292,14 @@ export function Player(props: PlayerProperties): JSX.Element {
         main_attrs.className += " " + combined.ui_class;
     }
 
+    if (viewReportContext && viewReportContext.reported.id === player_id) {
```

Comment:
This feels wrong, we shouldn't be spreading the context logic over to a generic component like `Player` I don't think. How about instead we put that context stuff up in the report code and then pass `"reported"` in for the `ui_class` for `Player`?